.gitignore

```diff
@@ -20,8 +20,7 @@ pids
 **/.DS_Store
 
 # CocoaPods
-/apps/macos/src/Pods/*
-/apps/ios/src/Pods/*
```

Comment:
Since this is generated, we usually want that to be ignored so people aren't spammed with files when they go to push changes

Reply:
The change I'm making is to ignore the more generic `**/Pods/*`, which maps to any pods subdirectory , as opposed to the more specific ones only for the mac and iOS test apps